proposals/lip-0018.md

```diff
@@ -209,18 +209,17 @@ A reclaim transaction is of the following form:
 
 A reclaim transaction, `tx`, must fulfil the following points to be valid:
 
-- Let `old_addr` be the address according to the old format for `tx.senderPublicKey`. Then, `old_addr` must be contained in `unregisteredAddresses`.
-- Let `old_balance` be the sum of all funds send to `old_addr` in Beddows. Then `tx.asset` must contain the property `amount`, and the number represented by `tx.asset.amount` must be equal to `old_balance`.
-- Let `new_addr` be the address of the new format for `tx.senderPublicKey` and let `new_balance` be the balance of `new_addr`. If there exists no account for `new_addr`, then `new_balance` is 0. Then `tx.fee` must be smaller or equal than `old_balance + new_balance`.
+- Let `old_addr` be the address according to the old format for `tx.senderPublicKey`. `old_addr` must be contained in `unregisteredAddresses`.
+- Let `old_balance` be the sum of all funds send to `old_addr` in Beddows. Then `tx.asset` must contain the property `amount`, and `tx.asset.amount` must be equal to `old_balance`.
+
+Notice that the base properties of the transaction also have to be valid. In particular, this implies that the transaction fee is paid by the account corresponding to `tx.senderPublicKey`. In some cases, users might need to credit accounts specifically to reclaim the balance sent to "unregistered" addresses.
```

Comment:
Suggestion:
```
Notice that the base properties of the transaction also have to be valid. In particular, this implies that the transaction fee is paid by the account corresponding to `tx.senderPublicKey`. Hence this account needs to exist beforehand and have enough balance to pay for the fee.
```

Reply:
I like Iker's suggestion. The current formulation of the last sentence would rather be suited for the rationale section. Also, isn't it rather "always" than "in some cases"?

Reply:
Changed to Iker's suggestion.

Reply:
Changed for Iker's suggestion.